pressbooks/pressbooks
# Replace default book cover images

This PR replaces the default cover image (red typewriter) with a new default cover image which matches the visual elements that were part of our recent rebrand. It fixes #2195. 

One note: these images are 2x larger than previous ones. Partially I think this is because the jpg has more information in it, but may also be because I don't have a copy of Adobe Illustrator to make web optimized jpg versions of these files from the .ai source file. If someone does have Illustrator and would like to do this to optimize image size, let me know ... Image can be found here: https://drive.google.com/file/d/11jyFqP3G2bHMqbXlrr0CQq_-RI_B6toB/view?usp=sharing

## To Test
1. Checkout this branch locally
2. Look at a previously created book with a default cover image. Check to see that the image has been replaced with the new image
3. Create a new book. Check to see that its cover image it the new default image
